src/terms.js

```diff
@@ -257,6 +259,7 @@ export default [
     'paradigms?( shift)?',
     'partner(ships?)?',
     'patents?',
+    'people are (saying|claiming|telling)'
```

Comment:
Missing `,`

Reply:
Thanks - serves me right for not using a linter. 

I'll close this and re-open a PR as I've no longer the source fork available